packages/fxa-auth-server/scripts/prune-tokens.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -e
+
+# We can set mode to dev here. Running with a value of production requires
+# more extensive configuration that is not necessary for the token pruner.
+export NODE_ENV=dev
+
+# Assumes shell script is executed from fxa-auth-server root folder.
+node -r esbuild-register ./scripts/prune-tokens.ts $@
```

Comment:
nit: could figure out the path of this script and use that to call the ts?   It would be a little nicer.

But the real concern here is that esbuild-register is a dev dependency so it won't be in the docker image.  If the purpose of this script is for SREs to run in stage or prod, then that can be an issue.